flagd/cmd/start.go

```diff
@@ -89,6 +90,7 @@ func init() {
 	flags.StringToStringP(headerToContextKeyFlagName, "H", map[string]string{}, "add key-value pairs to map "+
 		"header values to context values, where key is Header name, value is context key")
 	flags.Duration(streamDeadlineFlagName, 0, "Set a server-side deadline for flagd sync and event streams (default 0, means no deadline).")
+	flags.Bool(enableSyncContext, false, "Enable or disable sync context. Defaults to false.")
```

Comment:
```suggestion
	flags.Bool(enableSyncContext, false, "Enable or disables including static context values (see the -X flag) in the sync payload, instead of in the GetMetadataResponse. Defaults to false, but will be default to true in later versions.")
```